cloud/assembly.yml.lock

```diff
@@ -1,11 +1,29 @@
-install_repository_dependencies: false
-install_resolver_dependencies: false
-install_tool_dependencies: false
+install_repository_dependencies: 'true'
+install_resolver_dependencies: 'false'
+install_tool_dependencies: 'false'
```

Comment:
Was this intentional?

Reply:
The yaml dump that is creating the file is doing it by default, so I was going to try and see if it works so I don't have to change it, but it's an easy change if needed

Reply:
Ohhh the false to true was intentional because I had misunderstood what "repository_dep" meant (interpreted it as the scripts next to the wrapper) but it likely should stay false I think

Reply:
Yeah, this means that if this repo depends on other repos those other repos will also be installed, which you probably don't want.